packages/payment-processor/test/payment/encoder-approval.test.ts

```diff
@@ -228,6 +229,61 @@ const validRequestEthConversionProxy: ClientTypes.IRequestData = {
   },
 };
 
+export const validMetaRequest: ClientTypes.IRequestData = {
```

Comment:
Can't you re-use the same fixture for `encoder-payment.test.ts`?

Reply:
The data from both test are not the same. 
All the mocks should be shared in a common file, but this is already a big PR and I didn't want to include this mock refactoring as well. 
Sharing the data between both file would require to update the associated tests as the expected recipient / currencies as well as